packages/ruby/spec/readme/metrics_spec.rb

```diff
@@ -31,7 +31,10 @@
   end
 
   def app
-    SetHttpVersion.new(Readme::Metrics.new(noop_app, "API_KEY"))
+    app = Readme::Metrics.new(noop_app, "API_KEY") { |env|
+      {id: "id", label: "label", email: "email"}
```

Comment:
Thoughts on having a middleware in front of the app that sets these values so we can read them from the env rather than hardcoding? That might be a bit more realistic but maybe that's also overkill

Reply:
I'll try it out.